src/__tests__/components/Dashboard/DashboardCard.test.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Card from "../../../components/Dashboard/DashboardCard";
+
+describe('Dashboard card', () => {
+  let wrapper;
+  const props = {
+    height: '500px',
+    width: '500px',
+    title: 'Title Card'
+  };
+  beforeEach(() => {
+    wrapper = mount(<Card {...props} />);
+  });
+  afterEach(() => {
+    wrapper.unmount();
+  });
+  it('renders appropriately', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+  it('renders dashboard card with title if provided', () => {
+    const card = wrapper.find('.dashboard-card');
+    expect(card.length).toEqual(1);
+    expect(card.text()).toEqual(props.title);
```

Comment:
Nice work on the test. I think it would be better to test the actual value here that is `Title card` instead of props.title